README.md

```diff
@@ -17,17 +17,18 @@
   <hr />
 </p>
 
-The ForgeRock iOS SDK enables you to quickly integrate the [ForgeRock Identity Platform](https://www.forgerock.com/digital-identity-and-access-management-platform) into your iOS apps.
+The Ping SDk for iOS enables you to quickly integrate Ping products into your iOS apps.
 
-Use the SDKs to leverage _[Intelligent Authentication](https://www.forgerock.com/platform/access-management/intelligent-authentication)_ in [ForgeRock's Access Management (AM)](https://www.forgerock.com/platform/access-management) product, to easily step through each stage of an authentication tree by using callbacks.
+Use the SDKs to leverage _[Intelligent Access](https://www.pingidentity.com/en/platform/capabilities/intelligent-access.html)_ to easily step through each stage of an authentication tree by using callbacks.
 
 <!------------------------------------------------------------------------------------------------------------------------------------>
 <!-- REQUIREMENTS - Supported AM versions, API versions, any other requirements. -->
 
 ## Requirements
 
 * ForgeRock Identity Platform
-    * Access Management (AM) 6.5.2+
+  * Ping Advanced Identity Cloud
+  * PingAM 6.5.2+
 * iOS 12 and above
 * Xcode
 * Swift 5.x or Objective-C
```

Comment:
should we use this opportunity and remove Objective-C from here?

Reply:
removed